packages/rancher-kiali-server/generated-changes/overlay/templates/psp.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: kiali-psp
```

Comment:
@brendarearden I believe that Kiali uses templating for naming resources (as I saw [here](https://github.com/rancher/charts/blob/afb287748916cdf3af3e705f3d676a7d002f5f88/charts/rancher-kiali-server/rancher-kiali-server/1.29.000-rc01/templates/deployment.yaml#L5)), so we might need to change these to use helper templates instead of literal names. 

Not sure if the same applies to Istio, but I'm not familiar enough with either chart's naming conventions to know what the right names are to use.

Reply:
yes, we should also use the kiali name templating for the resource